tap_bing_ads/__init__.py

```diff
@@ -679,13 +727,19 @@ async def poll_report(client, account_id, report_name, start_date, end_date, req
 def log_retry_attempt(details):
     LOGGER.info('Retrieving report timed out, triggering retry #%d', details.get('tries'))
 
+# backoff when URLError occurs with time out
+@bing_ads_error_handling
 @backoff.on_exception(backoff.constant,
                       requests.exceptions.ConnectionError,
                       max_tries=5,
                       on_backoff=log_retry_attempt)
 def stream_report(stream_name, report_name, url, report_time):
     with metrics.http_request_timer('download_report'):
-        response = SESSION.get(url, headers={'User-Agent': get_user_agent()})
+        # Set request timeout with config param `request_timeout`.
+        # If value is 0,"0","" or not passed then set default to 300 seconds.
+        config_request_timeout = CONFIG.get('request_timeout')
+        timeout = config_request_timeout and  float(config_request_timeout) or REQUEST_TIMEOUT
```

Comment:
Use if-else instead of the ternary operation to keep consistency and avoid pylint error in the future.

Reply:
updated the logic to use if-else instead of ternary operation

Reply:
@namrata270998 It's looks like you missed this change. Can you use if-else to keep consistency?

Reply:
Right, missed it as it was present at 2 places
Updated it